scalafix-core/shared/src/main/scala/scalafix/internal/rule/RemoveXmlLiterals.scala

```diff
@@ -21,7 +21,10 @@ import scalafix.rule.RuleCtx
  * Xml patterns will not be supported by the xml interpolator,
  * until we know how to rule `case <a>{ns @ _*}</a>`.
  */
-case object RemoveXmlLiterals extends Rule("RemoveXmlLiterals") {
+case object RemoveXmlLiterals
+    extends Rule(
+      "RemoveXmlLiterals",
+      "Rule that removes XML literals to support Dotty's XML string interpolation.") {
```

Comment:
Rewrite that converts xml literals into interpolators for use with densh/scala-xml-quote